pkg/fanal/secret/builtin-rules.go

```diff
@@ -142,6 +142,14 @@ var builtinRules = []Rule{
 		Regex:    MustCompile(`ghr_[0-9a-zA-Z]{76}`),
 		Keywords: []string{"ghr_"},
 	},
+	{
+		ID:       "github_pat",
+		Category: CategoryGitHub,
+		Title:    "GitHub Fine-grained personal access tokens",
+		Severity: "CRITICAL",
+		Regex:    MustCompile(`^github_pat_[a-zA-Z0-9]{22}_[a-zA-Z0-9]{59}$`),
```

Comment:
I think this secret may be part of the line.
See example in #5649.

```suggestion
		Regex:    MustCompile(`github_pat_[a-zA-Z0-9]{22}_[a-zA-Z0-9]{59}`),
```